content/en/blog/_posts/2023-08-15-pkgs-k8s-io-introduction.md

```diff
@@ -0,0 +1,205 @@
+---
+layout: blog
+title: "pkgs.k8s.io: Introducing Kubernetes community-owned package repositories"
+date: 2023-08-15
```

Comment:
Release comms need to confirm the date here.

Reply:
@ramrodo was following up with me on this and can confirm if that's the target date. I can update it once we have written confirmation

Reply:
Currently, there are 2 1.28 feature blogs proposed to be published for August 15
- Kubernetes 1.28: Non-Graceful Node Shutdown Moves to GA
- pkgs.k8s.io: Introducing Kubernetes community-owned package repositories

Can the 1.28 comms team confirm the publication date 
@bradmccoydev @krol3 @mashby2022 @ramrodo @thschue

Reply:
Can we please put the date to 2023-08-22 please @xmudrii. Thanks

Reply:
Can we release earlier than Aug 15? The blog post is a dependency for the docs, mean we can't publish docs without the blog post.

Reply:
We can _merge_ the article earlier. I'm aiming to take care of both details (blog article and associated docs changes).

Reply:
Confirmed as 2023-08-15. Let's set the _time_ to be later than the official release announcement.

Reply:
Good, thanks!